File: src/main/java/com/seaweed/identifier/config/WebFilterConfig.java
package com.seaweed.identifier.config;

import jakarta.servlet.Filter;
import org.springframework.boot.web.servlet.FilterRegistrationBean;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class WebFilterConfig {

    @Bean
    public FilterRegistrationBean<Filter> loginFilter(){
        var filterRegistrationBean = new FilterRegistrationBean<Filter>();
        filterRegistrationBean.setFilter(new AuthFilter());
        filterRegistrationBean.setOrder(1);
        filterRegistrationBean.addUrlPatterns("/*");
        return filterRegistrationBean;
    }
}

File: src/main/kotlin/com/seaweed/identifier/config/WebFilterConfig.kt
package com.seaweed.identifier.config

import jakarta.servlet.Filter
import org.springframework.boot.web.servlet.FilterRegistrationBean
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class WebFilterConfig {

    @Bean
    fun loginFilter() : FilterRegistrationBean<Filter> {
        var filterRegistrationBean = FilterRegistrationBean<Filter>()
        filterRegistrationBean.filter = AuthFilter()
        filterRegistrationBean.order = 1;
        filterRegistrationBean.addUrlPatterns("/*")
        return filterRegistrationBean
    }
}